lib/matplotlib/colors.py

```diff
@@ -1225,6 +1240,594 @@ def reversed(self, name=None):
         return new_cmap
 
 
+class MultivarColormap(ColormapBase):
+    """
+    Class for holding multiple `~matplotlib.colors.Colormap` for use in a
+    `~matplotlib.cm.VectorMappable` object
+
+    MultivarColormap does not support alpha in the constituent
+    look up tables (ignored).
+    """
+    def __init__(self, name, colormaps, combination_mode):
+        """
+        Parameters
+        ----------
+        name : str
+            The name of the colormap family.
+        colormaps: list or tuple of `~matplotlib.colors.Colormap` objects
+            The individual colormaps that are combined
+        combination_mode: str, 'Add' or 'Sub'
+            Describe how colormaps are combined in sRGB space
+            'Add' -> additive
+            'Sub' -> subtractive
+        """
+        self.name = name
+
+        if not np.iterable(colormaps) or len(colormaps) == 1:
+            raise ValueError("A MultivarColormap must have more than one colormap.")
+        for cmap in colormaps:
+            if not issubclass(type(cmap), Colormap):
+                raise ValueError("colormaps must be a list of objects that subclass"
+                                 " Colormap, not strings or list of strings")
```

Comment:
We could easily add this functionality, with a lookup in the existing colormap registry.
I have been hesitant to include this because I suspect most combinations will result in multivariate colormaps that have poor mixing performance.
Nonetheless, allowing for strings will encourage more users to experiment with the functionality, and get a feel for how different colors mix.

Reply:
I added this functionality, and we will see about adding a tutorial later.